src/shape/shape.rs

```diff
@@ -24,17 +24,28 @@ impl<N: RealField, T: 'static + Shape<N> + Clone> ShapeClone<N> for T {
 ///
 /// This allows dynamic inspection of the shape capabilities.
 pub trait Shape<N: RealField>: Send + Sync + Downcast + ShapeClone<N> {
-    /// The AABB of `self`.
+    /// The AABB of `self` transformed by `m`.
     #[inline]
     fn aabb(&self, m: &Isometry<N>) -> AABB<N>;
 
-    /// The bounding sphere of `self`.
+    /// The AABB of `self`.
+    #[inline]
+    fn local_aabb(&self) -> AABB<N>;
```

Comment:
To avoid a breaking change:

```suggestion
    fn local_aabb(&self) -> AABB<N> {
        self.aabb(&Isometry::identity())
    }
```